### PR TITLE
[test] fix Thread node launch order

### PR DESCRIPTION
**Pull request by gjc13**
Chances are that the ot-cli-ftd node keeps receiving annoouncements
from the ot-cli-mtd node and cannot become a leader. Modify the launch
order to resolve this issue.

Fixes https://github.com/openthread/ot-br-posix/issues/569

### CODE REVIEW

**Review by bukepo — APPROVED** [review]
LGTM:+1:

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1: 

I think this was due to slow Github Runner VM.

**Comment by librasungirl** [comment]
@gjc13 why `force` could improve the stability? how is the stability proved to be improved?

**Comment by gjc13** [comment]
> @gjc13 why `force` could improve the stability? how is the stability proved to be improved?

Don't really know why but our observation is that the ot-cli-ftd node is still in detached mode after 5 seconds so I added a force command to make it a leader.

**Comment by librasungirl** [comment]
@gjc13 
It might not be expected to issue similar force command for physical devices.
should we check the root cause for the device not becoming leader, other than workaround to make this test pass?

**Comment by gjc13** [comment]
> @gjc13
> It might not be expected to issue similar force command for physical devices.
> should we check the root cause for the device not becoming leader, other than workaround to make this test pass?

Just identified and fixed the root cause.

File: tests/dbus/test-client [hunk 1]
```diff
@@ -99,7 +88,22 @@ send "commissioner joiner add * ABCDEF\r\n"
 expect "Done"
 wait
 EOF
-    sleep 15
+    sleep 5
+    expect <<EOF &
```
Comment (librasungirl) [inline tests/dbus/test-client:92]: nit: better to add comment to explain why introducing the `mtd 2` here for readability.

**Review by librasungirl — APPROVED** [review]

File: tests/dbus/test-client [hunk 2]
```diff
@@ -63,20 +66,7 @@ main()
...
-expect "Done"
-send "channel 11\r\n"
-expect "Done"
-send "ifconfig up\r\n"
-expect "Done"
-send "thread start\r\n"
-expect "Done"
-wait
-EOF
+    # The ot-cli-ftd node is used for test Thread join.
```
Comment (jwhui) [inline tests/dbus/test-client:69]:
```suggestion
    # The ot-cli-ftd node is used to test Thread attach.
```

File: tests/dbus/test-client [hunk 3]
```diff
@@ -99,7 +89,23 @@ send "commissioner joiner add * ABCDEF\r\n"
 expect "Done"
 wait
 EOF
-    sleep 15
+    sleep 5
+    # The ot-cli-mtd node is used for test the child and neighbor table.
```
Comment (jwhui) [inline tests/dbus/test-client:93]:
```suggestion
    # The ot-cli-mtd node is used to test the child and neighbor table.
```

**Review by jwhui — APPROVED** [review]

**Pull request merged**